src/users/create_or_modify/v2.rs

```diff
@@ -90,3 +96,9 @@ impl Response {
         Self { details }
     }
 }
+
+/// An external ID associated with a user
+// TODO
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+pub struct IncomingExternalId {}
```

Comment:
This should not be needed. Did you previously have an `ExternalId` type with a lifetime parameter?

Reply:
Yes indeed, that's my bad, I removed it.